content/_help/verify-your-identity/phone-number_en.md

```diff
@@ -1,22 +1,18 @@
 ---
```

Comment:
I think the build failure may be due to missing periods in each of the file names.

- `phone-number_en.md` should be `phone-number._en.md`
- `phone-number_es.md` should be `phone-number._es.md`
- `phone-number_fr.md` should be `phone-number._fr.md`

Reply:
Thank you!!!